Movieee/Movieee/UIs/Genre Category Detail Screen/GenreCateScreenViewController.swift

```diff
@@ -0,0 +1,101 @@
+import UIKit
+
+private enum GenreCateConstraints {
+    static let sizeOfItem = CGSize(width: 175, height: 300)
+    static let idGenreCateCollectionCell = "GenreCateCollectionCell"
+}
+
+enum ViewType {
+    case genre
+    case category
+    func configTitle(genre: String, category: String) -> String {
+        return self == .genre ? genre : category
+    }
```

Comment:
```suggestion
    var title: String {
        switch self: {
            case .genre:
                return genreInfo?.name.uppercased() ?? ""
            case .category:
                return categoryName
        }
    }
```
để enum này bên trong VC cho dễ dùng